app/experimenter/nimbus-ui/src/components/AppLayoutWithExperiment/index.tsx

```diff
@@ -88,7 +83,7 @@ const AppLayoutWithExperiment = ({
     };
   }, [startPolling, stopPolling, experiment, polling]);
 
-  if (experimentLoading || analysisRequired) {
+  if (experimentLoading || (analysisRequired && analysisLoading)) {
```

Comment:
Hmmmm, I think you will still get a flash of content here. `experimentLoading` becomes false ever so slightly before `analysisLoading` becomes true. You can see this locally by throttling the Network tab to something like 3G, but I assume this would be more visible once deployed. 

It's almost like we need a way for `analysisLoading` to _start_ true. Or we need a way to do `(experimentLoading || (analysisRequired && analysisRequestComplete))` to check if the request was finished (which would be irrelevent in renders where analysisRequired is false). 🤔